README.md

```diff
@@ -2,6 +2,15 @@
 
 Discussion agents.
 
+# Windows installation 
+```
+1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console
+2.conda create -n my_env python=3.10.13
+3.conda activate my_env
+4.pip install pipx
+5.pipx install poetry (make sure to add poetry to path by adding C:\Users\<username\.local\bin to path in advanced system settings)
```

Comment:
Missing ">" sign after "username". Probably want to go into detail about adding the path.